region/tourian/main/Metroid Room 1.json

```diff
@@ -995,6 +1028,57 @@
         "Then use the frozen Rinka to cross the acid."
       ]
     },
+    {
+      "link": [4, 1],
+      "name": "Metroid Room 1 Rinka Boost (Right to Left)",
+      "notable": true,
+      "requires": [
+        {"enemyDamage": {
+          "enemy": "Rinka",
+          "type": "contact",
+          "hits": 1
+        }},
+        "canTrickyJump",
+        "canHorizontalDamageBoost",
+        {"or": [
+          {"obstaclesCleared": ["A"]},
+          {"and": [
+            "canMetroidAvoid",
+            "canHitbox"
+          ]}
+        ]},
+        {"or": [
+          "canInsaneJump",
+          {"enemyDamage": {
+            "enemy": "Rinka",
+            "type": "contact",
+            "hits": 1
+          }},
+          {"and": [
+            "canTrickyJump",
+            "canLateralMidAirMorph"
+          ]},
+          {"and": [
+            "canTrickyJump",
+            "canPreciseWalljump",
```

Comment:
isnt this option the one to get to the left platform, which the damage boost is helping doing instead? 
Is the second rinka option using the same rinka? do you need to worry about it going off screen?
Do these rinka damage boost strats need `canCameraManip`?

Reply:
- The idea is to walljump against the left wall of the acid pool.  It does look similar to the walljump up to the floating platform.  I think I'll keep it because Rinka damage can be less than the acid damage if Gravity reduces contact damage but not environmental.  So there would still be a way to this branch.
- Using the same rinka makes the dboosts a good bit harder, so I'll cut it since you can do the jump normally at that point.  It going offscreen is a big concern when doing two boosts.
- Added `canCameraManip`